tools/merge_licenses.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -o xtrace
+set -o errexit
+set -o nounset
+set -o pipefail
+
+output_path=${BUILD_WORKING_DIRECTORY}/MERGED_LICENSES
+# Truncate file.
+echo >$output_path
+
+for path in $(find ${BUILD_WORKING_DIRECTORY}/vendor/ -name LICENSE); do
+    echo adding $path;
```

Comment:
quote entire log message, echo to stderr.

Reply:
Done